docs/plugins/introduction.md

```diff
@@ -0,0 +1,19 @@
+# About plugins
```

Comment:
NB: Naming it like this to ensure this will be the first doc listed on the website, since they go in alphabetical order

...Probably room for improvement to avoid needing such hack 🤓

Reply:
We need to use a better DOC framework, imo. Too many restrictions, page ordering, max depth, etc ...